tests/api.v2/controllers/plotController.test.js

```diff
@@ -0,0 +1,68 @@
+// @ts-nocheck
```

Comment:
what does `@ts-nocheck` mean?

Reply:
This disables semantic checks because the linter doesn't recognize jest mocks